composer.json

```diff
@@ -29,19 +29,22 @@
         }
     },
     "require": {
-        "php": "^5.6 || ^7.0"
+        "php": "^5.6 || ^7.0",
+        "psr/container": "^1.0"
     },
     "require-dev": {
         "phpunit/PHPUnit": "^5.6",
         "athletic/athletic": "^0.1",
         "zendframework/zend-stdlib": "^2.7.3 || ^3.0",
-        "container-interop/container-interop": "^1.1.0",
         "zendframework/zend-coding-standard": "~1.0.0"
     },
     "suggest": {
-        "container-interop/container-interop": "^1.1.0, to use the lazy listeners feature",
+        "psr/container": "^1.0.0, to use the lazy listeners feature",
```

Comment:
@froschdesign good catch - should have been `require-dev` (not `require`) and `suggest`. Fixed.Thanks!